homeassistant/components/ampio/air_quality.py

```diff
@@ -0,0 +1,102 @@
+"""
+Support for Ampio Air Quality data.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/air_quality/ampio/
```

Comment:
We don't include the url in the docstring anymore. Just keep the very first line.

Reply:
Removed